example/docker-compose.yml

```diff
@@ -59,6 +59,18 @@ services:
       - "5601:5601"
     depends_on:
       - elasticsearch
+  dashboard-importer:
+    image: appropriate/curl
+    environment:
+      ELASTIC_PASSWORD: ${ELASTIC_PASSWORD}
+    volumes:
+      - type: bind
+        source: ../everything.ndjson
```

Comment:
doesn't look like this ndjson has been added to the commit? Probably replaces everything.json @ewancolyer

Reply:
yep 👍 I'm going to tidy up the dashboards and export them today

Reply:
When you do, overwrite/remove the current everything.json

Reply:
This has now been done